wg-lts/annual-report-2023.md

```diff
@@ -5,23 +5,23 @@
 
 1. What work did the WG do this year that should be highlighted?
 
-<!--
-   Some example items that might be worth highlighting:
-   - artifacts
-   - reports
-   - white papers
-   - work not tracked in KEPs
--->
+During 2023, we worked to restart the WG.
```

Comment:
Can you also please add notes from the discussion that happened during KCS EU 23?

Reply:
Here are the notes for the discussion at KCS EU 2023: https://docs.google.com/document/d/1C7utWMjGpDcBc5TB_ooddERsid59sFLC7gqLzyS5aKY/edit?usp=sharing